dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -50,7 +50,7 @@ CWPKI2006I=CWPKI2006I: The ACME certificate authority at the {0} URI provided th
 CWPKI2006I.explanation=The certificate authority provides terms of service.
 CWPKI2006I.useraction=Review the provided terms of service.
 
-CWPKI2007I=CWPKI2007I: The ACME service installed a certificate with a {0} serial number that is signed by the ACME certificate authority at the {1} URI. The expiration date is {2}.
+CWPKI2007I=CWPKI2007I: The ACME service installed a certificate with {0} serial number that is signed by the ACME certificate authority at the {1} URI. The expiration date is {2}.
```

Comment:
change
The ACME service installed a certificate with {0} serial number 

to
The ACME service installed a certificate with the {0} serial number